webapp/components/generic/SearchPost/SearchPost.spec.js

```diff
@@ -0,0 +1,64 @@
+import { shallowMount } from '@vue/test-utils'
+import SearchPost from './SearchPost.vue'
+
+const localVue = global.localVue
+localVue.filter('dateTime', d => d)
+
+describe('SearchPost.vue', () => {
+  let mocks, wrapper, propsData
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(string => string),
+    }
+    propsData = {
+      option: {
+        title: 'Post Title',
+        commentsCount: 3,
+        shoutedCount: 6,
+        createdAt: '23.08.2019',
+        author: {
+          name: 'Post Author',
+        },
+      },
+    }
+    wrapper = Wrapper()
+  })
+
+  const Wrapper = () => {
+    return shallowMount(SearchPost, { mocks, localVue, propsData })
+  }
+
+  describe('shallowMount', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-08T15:12:47Z" title="Wednesday, January 8th 2020, 4:12:47 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,60 @@
+import { shallowMount } from '@vue/test-utils'
+import SearchPost from './SearchPost.vue'
+
+const localVue = global.localVue
+localVue.filter('dateTime', d => d)
+
+describe('SearchPost.vue', () => {
+  let mocks, wrapper, propsData
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(string => string),
+    }
+    propsData = {
+      option: {
+        title: 'Post Title',
+        commentsCount: 3,
+        shoutedCount: 6,
+        createdAt: '23.08.2019',
+        author: {
+          name: 'Post Author',
+        },
+      },
+    }
+    wrapper = Wrapper()
+  })
+
+  const Wrapper = () => {
+    return shallowMount(SearchPost, { mocks, localVue, propsData })
+  }
+
+  describe('mount', () => {
```


Shouldn’t that be `shallowMount`?